src/app/view/dashboard/project/project-team/project-team.component.spec.ts

```diff
@@ -79,12 +79,98 @@ describe('ProjectTeamComponent', () => {
+            const mockProjectsJson = {
+                'projects': [{
+                    'shortname': 'biblio',
+                    'description': 'Bibliography',
+                    'institution': null,
+                    'logo': null,
+                    'dataNamedGraph': 'http://www.knora.org/data/biblio',
+                    'id': 'http://data.knora.org/projects/DczxPs-sR6aZN91qV92ZmQ',
+                    'status': true,
+                    'selfjoin': false,
+                    'keywords': null,
+                    'longname': 'Bibliography',
+                    'ontologyNamedGraph': 'http://www.knora.org/ontology/biblio'
```

Comment:
You will need to change the test data: `ontologyNamedGraph` is now `ontologies`

Reply:
I will fix it